docs/samples/advanced_config/aggregator.yaml

```diff
@@ -162,3 +162,31 @@ garbage_collection:
   # The maximum number of collection jobs (& related artifacts), per task, to delete in a single run
   # of the garbage collector.
   collection_limit: 50
+
+# Configuration for key rotator. Allows running the key rotator as part of the aggregator process.
+# If omitted, you should run the key rotator as a separate cronjob.
+key_rotator:
+  # How frequently to run the key rotator, in seconds. Required.
+  frequency_s: 3600
+
+  # Rotation policy for global HPKE keys. At least one is required. Each entry
+  # represents a key with a particular ciphersuite.
```

Comment:
As in https://github.com/divviup/janus/pull/3136/files#r1635380069, I think the second and third sentences belong on the `ciphersuite` key now.